extensions/Worker.Extensions.Http.AspNetCore/src/Coordinator/ContextReference.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Coordinator
```

Comment:
This namespace seems off. Can we drop `Coordinator` here?

Reply:
Yes -- VS auto-updated these when I organized them into folders and I didn't notice. @satvu mentioned the same thing.